tests/test_clean.py

```diff
@@ -372,73 +605,77 @@ def test_svg_allow_local_href_nonlocal(text, expected):
     ATTRS = {
         'pattern': ['id', 'href'],
     }
-    assert bleach.clean(text, tags=TAGS, attributes=ATTRS) == expected
+    assert clean(text, tags=TAGS, attributes=ATTRS) == expected
+
+
+@pytest.mark.xfail(reason='regression from bleach 1.4')
+def test_weird_strings():
+    s = '</3'
+    assert clean(s) == '</3'
 
 
-@pytest.mark.xfail(reason='html5lib >= 0.99999999: changed API')
+@pytest.mark.xfail(reason='regression from bleach 1.4')
 def test_sarcasm():
     """Jokes should crash.<sarcasm/>"""
-    dirty = 'Yeah right <sarcasm/>'
-    clean = 'Yeah right &lt;sarcasm/&gt;'
-    assert bleach.clean(dirty) == clean
-
+    assert (
+        clean('Yeah right <sarcasm/>') ==
+        'Yeah right &lt;sarcasm/&gt;'
+    )
```

Comment:
I inlined the text here to make it easier to read. I also fixed the xfail reason comment.